documentation/specs/src/base-ledger/multisignature.md

```diff
@@ -1,3 +1,114 @@
-## k-of-n multisignature
+# k-of-n multisignature
 
-The k-of-n multisignature validity predicate authorises transactions on the basis of k out of n parties approving them.
\ No newline at end of file
+The k-of-n multisignature validity predicate authorizes transactions on the basis of k out of n parties approving them. This document targets the encrypted wasm transactions: at the moment there's no support for multisignature on wrapper or protocol transactions.
```

Comment:
We do not need multisignatures for wrapper or protocol transactions, as far as I am aware